### PR TITLE
Context manager to handle redis connections along with pottery demo 

### PR DESCRIPTION
**Pull request by why-not-try-calmer**

### CODE REVIEW

File: defrag/modules/db.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+from os import environ as env
+from typing import Optional, Union
+from redis import Redis, BlockingConnectionPool
+from redis.client import Pipeline
+from pottery import RedisDict
+
+__MOD_NAME__ = "db"
```
Comment (KaratekHD) [inline defrag/modules/db.py:7]: I do not want to have things like this as a module. I'd suggest creating a `modules/db` folder and placing it in there, or putting it in `helpers` or `utils`

**Review by KaratekHD — CHANGES_REQUESTED** [review]

File: defrag/modules/db.py (hunk 1 above)
Comment (why-not-try-calmer) [reply defrag/modules/db.py]: did that

**Comment by why-not-try-calmer** [comment]
When it's time to merge this PR it should be merged using the `Squash and merge` option from the command button.

**Comment by why-not-try-calmer** [comment]
Making this into a Draft PR as it's moving far beyond just adding pottery.

**Pull request closed without merging**